lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgstr "Senaste utskrift"
+
+# MSG_SELFTEST_EXTRUDER_FAN c=20
+#: messages.c:89
+msgid "Left hotend fan?"
+msgstr "V\xe1nst hotend fl\xe1kt?"
+
+# MSG_LEFT c=10
+#: ultralcd.cpp:2844
+msgid "Left"
+msgstr "V\xe1nster"
+
+# MSG_BED_CORRECTION_LEFT c=14
+#: ultralcd.cpp:3114
+msgid "Left side [um]"
+msgstr "V\xe1nstsida [um]"
+
+# MSG_LIN_CORRECTION c=18
+#: ultralcd.cpp:5702
+msgid "Lin. correction"
+msgstr "Linj\xe1r korrektion"
+
+# MSG_BABYSTEP_Z c=18
+#: messages.c:12
+msgid "Live adjust Z"
+msgstr "Live justera Z"
+
+# MSG_INSERT_FIL c=20 r=6
+#: ultralcd.cpp:7380
+msgid "Insert the filament (do not load it) into the extruder and then press the knob."
+msgstr "Infoga filamentet (ladda inte in det) i extrudern och tryck sedan pa knappen."
+
+# MSG_LOAD_FILAMENT c=17
+#: messages.c:58
+msgid "Load filament"
+msgstr "Ladda filament"
+
+# MSG_LOADING_COLOR c=20
+#: ultralcd.cpp:2516
+msgid "Loading color"
+msgstr "Laddar f\xe1rg"
+
+# MSG_LOADING_FILAMENT c=20
+#: messages.c:59
+msgid "Loading filament"
+msgstr "Laddar filament"
+
+# MSG_ITERATION c=12
+#: messages.c:53
+msgid "Iteration"
+msgstr "Iteration"
+
+# MSG_LOOSE_PULLEY c=20
+#: ultralcd.cpp:8167
+msgid "Loose pulley"
+msgstr "Lossa pulley"
+
+# MSG_LOAD_TO_NOZZLE c=18
+#: ultralcd.cpp:6717
+msgid "Load to nozzle"
+msgstr "Ladd till munstyck"
+
+# MSG_M117_V2_CALIBRATION c=25
+#: messages.c:62
+msgid "M117 First layer cal."
+msgstr "M117 F\xefrst lager kalibr."
+
+# MSG_MAIN c=18
+#: messages.c:63
+msgid "Main"
+msgstr "Huvudsaklig"
+
+# MSG_BL_HIGH c=12
+#: messages.c:155
+msgid "Level Bright"
+msgstr "Ljusniva"
+
+# MSG_BL_LOW c=12
+#: messages.c:156
+msgid "Level Dimmed"
+msgstr "Niva d\xe1mpad"
+
+# MSG_MEASURE_BED_REFERENCE_HEIGHT_LINE1 c=20 r=3
+#: messages.c:67
+msgid "Measuring reference height of calibration point"
+msgstr "M\xe1tning av referensh\xefjd f\xefr kalibreringspunkt"
+
+# MSG_MESH_BED_LEVELING c=18
+#: messages.c:148
+msgid "Mesh Bed Leveling"
+msgstr "B\xe1dd nivautj\xe1mning"
+
+# MSG_MMU_OK_RESUMING_POSITION c=20 r=4
+#: mmu.cpp:765
+msgid "MMU OK. Resuming position..."
+msgstr "MMU OK. Aterupptar position..."
+
+# MSG_MMU_OK_RESUMING_TEMPERATURE c=20 r=4
+#: mmu.cpp:758
+msgid "MMU OK. Resuming temperature..."
+msgstr "MMU OK. Aterupptar temperatur..."
+
+# MSG_MEASURED_SKEW c=14
+#: ultralcd.cpp:2885
+msgid "Measured skew"
+msgstr "M\xe1tt skevhet"
+
+# MSG_MMU_FAILS c=15
+#: messages.c:69
+msgid "MMU fails"
+msgstr "MMU felar"
+
+# MSG_MMU_LOAD_FAILED c=20
+#: mmu.cpp:1587
+msgid "MMU load failed"
+msgstr "MMU-laddning felade"
+
+# MSG_MMU_LOAD_FAILS c=15
+#: messages.c:70
+msgid "MMU load fails"
+msgstr "MMU-laddn felar"
+
+# MSG_MMU_OK_RESUMING c=20 r=4
+#: mmu.cpp:776
+msgid "MMU OK. Resuming..."
+msgstr "MMU OK. Aterupptar..."
+
+# MSG_MODE c=6
+#: messages.c:103
+msgid "Mode"
+msgstr "MMU Mode"
+
+# MSG_MK3_FIRMWARE_ON_MK3S c=20 r=4
+#: Marlin_main.cpp:882
+msgid "MK3 firmware detected on MK3S printer"
+msgstr "MK3-firmware uppt\xe1ckt pa MK3S-skrivare"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.